tests/test_create_dataset_manifest.py

```diff
@@ -346,6 +346,37 @@ def test_multitask(self):
         assert _get_instance_count_per_class(few_shot_manifest) == {2: 13, 3: 13, 0: 11, 4: 11, 1: 10, 5: 10}
 
 
+class TestManifestSubsetByRatio(unittest.TestCase):
```

Comment:
should we add a test for detection as well?

Reply:
good catch. added it.